synthesizer/process/src/lib.rs

```diff
@@ -170,9 +236,24 @@ impl<N: Network> Process<N> {
         lap!(timer, "Load circuit keys");
 
         // Add the stack to the process.
-        process.add_stack(stack);
+        process.credits = Some(Arc::new(stack));
+
+        finish!(timer, "Process::load_testing_only");
```

Comment:
same note on the name of the function here

Reply:
01d9a0b49